README.md

```diff
@@ -4,10 +4,10 @@
 [![Code Climate](https://codeclimate.com/github/cfpb/cfgov-refresh.png?branch=flapjack)](https://codeclimate.com/github/cfpb/cfgov-refresh?branch=flapjack)
 
 The in-progress redesign of the [consumerfinance.gov](http://consumerfinance.gov) website.
-This project includes the front-end assets and build tools,
+This Django project includes the front-end assets and build tools,
 [Jinja templates](http://jinja.pocoo.org) for front-end rendering,
 and [Sheer](https://github.com/cfpb/sheer) configurations for loading content from the
```

Comment:
Sheer is now just for indexing the data into ES. Django-sheerlike is actually doing the loading.